aya/src/maps/array/array.rs

```diff
@@ -39,7 +39,7 @@ impl<T: Borrow<MapData>, V: Pod> Array<T, V> {
         let data = map.borrow();
         check_kv_size::<u32, V>(data)?;
 
-        let _fd = data.fd_or_err()?;
+        let _fd = data.fd;
```

Comment:
This can go now? We used to get the fd to ensure that the map was created, but
now it's always created

Reply:
Done. Thanks for catching that.